### PR TITLE
address issue#392

### PR DESCRIPTION
**Pull request by weiwang-gsa**
- move ordering guide content to be under buyer's guide section
- add ordering guide to buyer's guide menu item
- adding ordering guide to side nav of buyer's guide section
- handle redirect

Note: seems redirect didn't work properly on feature branch, need to test if it works well after merging to staging branch.

### CODE REVIEW

File: src/config/redirects.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Configure a list of redirects.
+ */
+
+import path from "path"
+
+const redirects = {
+  "/ordering-guide/": "/buyers-guide/ordering-guide/",
```
Comment (mark-meyer) [inline src/config/redirects.js:8]: I think we need to give this some thought given that the site is being served at `https://gsa.gov/oasis-plus/`

Maybe adding a line: 
`"/oasis-plus/ordering-guide/": "/oasis-plus/buyers-guide/ordering-guide/",` 

will work?

Comment (weiwang-gsa) [reply src/config/redirects.js:8]: my initial checkin commit is using ""/ordering-guide/": "/oasis-plus/buyers-guide/ordering-guide/", but it both not working locally or in feature branch. I can revert it back for testing purpose though, I think feature branch is not rendering properly.

**Comment by weiwang-gsa** [comment]
As requirement changed, this PR is no longer needed, closed it.

**Comment by weiwang-gsa** [comment]
As requirement changed, this PR is no longer needed, closed it.

**Pull request closed without merging**